package.json

```diff
@@ -10,8 +10,8 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint:js": "eslint --ignore-path .gitignore --ext .js,.jsx i",
-    "lint:css": "prettier **/*.{json,scss,css}",
+    "lint:js": "eslint --ignore-path .gitignore --ext .jsx --ext .js .",
```

Comment:
You can separate extensions with a comma.

Reply:
It didn't work for me. Could you please post code sample how it should look like?

Reply:
This is from next. I think it works
```
eslint --ignore-path .gitignore --ext .js,.jsx .
```